app/models/container_template.rb

```diff
@@ -12,4 +12,55 @@ class ContainerTemplate < ApplicationRecord
   serialize :objects, Array
 
   acts_as_miq_taggable
+
+  def instantiate(params, project = nil)
+    @ems = ext_management_system
+    @clients = {}
+    project ||= container_project.name
+
+    temp = {
+      :metadata   => {
+        :name      => name,
+        :namespace => project
+      },
+      :objects    => objects,
+      :parameters => params
+    }
+
+    processed_template = process(@ems.connect, temp)
+    create_objects(processed_template['objects'], project) if processed_template
+  end
+
+  def process(client, temp)
+    client.process_template(temp)
+  rescue KubeException => e
+    $log.error("Unexpected Exception while processing template: #{e}")
```

Comment:
When logging use `_log`.  But I suspect this logging will be removed based on the previous comment.